src/RealtimeChannel.ts

```diff
@@ -406,13 +406,18 @@ export default class RealtimeChannel {
   ): RealtimeChannel {
     return this._on(type, filter, callback)
   }
-
+  /**
+   * Sends a message into the channel.
+   * @param args Arguments to send to the channel. Composed of a type, event, and payload.
```

Comment:
See https://stackoverflow.com/questions/6460604/how-to-describe-object-arguments-in-jsdoc

Reply:
That's the proper way to document an argument that is an object with it's fields